server/utils.js

```diff
@@ -28,17 +34,26 @@ var convertLatLngToStreet = function(lng, lat, callback) {
 
 var convertDirectionsToStreet = function(req, callback) {
   var responseObj = req.body.streets;
```

Comment:
Not a big deal but consider changing this variable name to something more descriptive